### DISCUSSION TITLE
Precursor-Succesor-ALM

### DISCUSSION THREAD

**Discussion by Nakhchi** [xcompact3d/Incompact3d#308] [Q&A] [answered]
Hello,

I wanted to ask about the possibility of running Precursor-Successor wind farm simulation based on ALM. Does the current code have this option? Precursor-Successor-ADM works well.

Many thanks,
Mahdi

**Comment by slaizet** [comment]
Hi,
Have you looked in https://github.com/xcompact3d/Incompact3d/tree/master/examples/Wind-Turbine/precursor-succesor-ADM?
Best,
Sylvain

**Reply by Nakhchi** [reply]
Hi Sylvain,

Yes. It works well with ADM. I want to know if it works with ALM as well. 

Thank you 
Mahdi

**Reply by slaizet** [reply, marked as answer]
Yes it does, see for instance https://www.sciencedirect.com/science/article/pii/S0142727X23000644
Best,
Sylvain
